frameworks/cassandra/tests/test_backup_and_restore.py

```diff
@@ -106,3 +113,98 @@ def test_backup_and_restore_to_s3() -> None:
         plan_parameters,
         config.get_foldered_node_address(),
     )
+
+
+@pytest.mark.aws
+@pytest.mark.sanity
+def test_backup_and_restore_to_s3_compatible_storage() -> None:
+    try:
+        sdk_install.install(
+            "minio",
+            "minio",
+            expected_running_tasks=0,
+            package_version="0.0.13-RELEASE.2018-10-06T00-15-16Z",
+            wait_for_deployment=False,
+        )
+        temp_key_id = os.getenv("AWS_ACCESS_KEY_ID")
+
+        if not temp_key_id:
+            assert (
+                False
+            ), 'AWS credentials are required for this test. Disable test with e.g. TEST_TYPES="sanity and not aws"'
+        temp_secret_Access_key = os.getenv("AWS_SECRET_ACCESS_KEY")
+        is_strict = sdk_utils.is_strict_mode()
+        if is_strict:
```

Comment:
this seems to be only place `is_strict` used, why not call the method `sdk_utils.is_strict_mode()` directly?